agent/src/main/scala/za/co/absa/atum/agent/AtumContext.scala

```diff
@@ -142,7 +141,7 @@ class AtumContext private[agent] (
   /**
    * Adds a measure to the AtumContext.
    *
-   * @param measure the measure to be added
+   * @param newMeasure the measure to be added
```

Comment:
`measure` is better then `newMeasure`, it's obvious from the context that it's a new measure. I suggest to rename the parameter to `measure`.

Reply:
I wonder if we could use vals instead of vars in AtumContext ...